scripts/erc20_to_native/sendHome.js

```diff
@@ -35,9 +35,9 @@ async function main() {
         privateKey: USER_ADDRESS_PRIVATE_KEY,
         data: '0x',
         nonce,
-        gasPrice: '1',
+        gasPrice: '1000000000',
         amount: HOME_MIN_AMOUNT_PER_TX,
```

Comment:
Could we make sure that `HOME_MIN_AMOUNT_PER_TX` is greater than the "Minimum amount per transaction" limit before starting to send transactions?

Reply:
Done. Added that check on every `sendHome/Foreign` script